common_protos/scaii.proto

```diff
@@ -215,3 +216,15 @@ message SerializationResponse {
 	required bytes serialized = 1;
 	required SerializationFormat format = 2;
 }
+// Since Rpc waits for a response for each packet that is sent,
+// but in non-testing mode, Viz does not send back state,
+// we send back a UserCommand of the NONE variety as a simple Ack.  
+// EXPLAIN is a standin for when explanation info is requested.
+enum UserCommandType {
+	NONE = 0;
+	EXPLAIN = 1;
+}
+message UserCommand {
+	required UserCommandType user_command_type = 1;

```

Comment:
maybe shorten this to just `command_type`? This name is a bit long and leaving off `user` doesn't make it any less clear

Reply:
will do